### PR TITLE
AP_RangeFinder: define 3_ADDR when we have three rangefinders

### PR DESCRIPTION
**Pull request by peterbarker**
Without this patch this field would only be defined when 4 rangefinders
were specified

### CODE REVIEW

File: libraries/AP_RangeFinder/RangeFinder.cpp [hunk 1]
```diff
@@ -227,6 +227,14 @@ const AP_Param::GroupInfo RangeFinder::var_info[] = {
...
 
 #if RANGEFINDER_MAX_INSTANCES > 2
+    // @Param: 3_ADDR
+    // @DisplayName: Bus address of third rangefinder
+    // @Description: This sets the bus address of the sensor, where applicable. Used for the LightWare I2C sensor to allow for multiple sensors on different addresses. A value of 0 disables the sensor.
+    // @Range: 0 127
+    // @Increment: 1
+    // @User: Advanced
+    AP_GROUPINFO("3_ADDR", 36, RangeFinder, _address[2], 0),
+
```
Comment (OXINARF) [inline libraries/AP_RangeFinder/RangeFinder.cpp]: This is nitpicking but is there any reason for putting this at the beginning and not at the end of the _if_ like the others?
We should also move the *_ADDR* and *2_ADDR* to be together with their respective parameters

Comment (magicrub) [inline libraries/AP_RangeFinder/RangeFinder.cpp]: you can't change the index and I do not suggest we move and store them out of order; it's just asking for trouble.

Comment (OXINARF) [inline libraries/AP_RangeFinder/RangeFinder.cpp]: The index was not changed and we have several out of order parameters so I don't think it is an issue here

Comment (peterbarker) [inline libraries/AP_RangeFinder/RangeFinder.cpp]: @OXINARF @magicrub Rearranged as suggested.

File: libraries/AP_RangeFinder/RangeFinder.cpp [hunk 2]
```diff
@@ -300,17 +301,19 @@ const AP_Param::GroupInfo RangeFinder::var_info[] = {
...
     // @Range: 0 127
     // @Increment: 1
     // @User: Advanced
     AP_GROUPINFO("3_ADDR", 36, RangeFinder, _address[2], 0),
-    
+
+#endif
+
+#if RANGEFINDER_MAX_INSTANCES > 3
+
```
Comment (lucasdemarchi) [inline libraries/AP_RangeFinder/RangeFinder.cpp]: while at it, fix the "@DisplayName" below?

Comment (OXINARF) [inline libraries/AP_RangeFinder/RangeFinder.cpp]: And above :smile:

Comment (peterbarker) [inline libraries/AP_RangeFinder/RangeFinder.cpp]: @lucasdemarchi @OXINARF ... and all through the entries :-)

Pushed a commit on top.

**Comment by lucasdemarchi** [comment]
Applied, thanks!

**Pull request closed without merging**